### PR TITLE
explain how to use comments

### PR DESCRIPTION
**Pull request by limratana**
and that # cannot be used in variable names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,15 @@ Whenever your application loads, these variables will be available in `ENV`:
 config.fog_directory  = ENV['S3_BUCKET']
 ```
 
+Comments may be added to your file as such:
+
+```comments
```
Comment (bkeepers) [inline README.md]: This should be `shell`.

**Comment by bkeepers** [comment]
Thanks for the pull request. It might be worth noting that `#` can be used in a string if it is wrapped in quotes (I think, we'll want to verify that):

```shell
FOO="String with # in the middle"
```

File: README.md (hunk 1 above)
Comment (limratana) [inline README.md]: I tested putting # in the middle of a string, it works. I am not sure what you mean by shell.

File: README.md [hunk 2]
```diff
@@ -101,6 +101,15 @@ Whenever your application loads, these variables will be available in `ENV`:
...
+#here is a comment
+SECRET_KEY=YOURSECRETKEYGOESHERE # comment
+```
+
+Variable names may not contain the # symbol, however values can.
```
Comment (paulcsmith) [inline README.md]: This doesn't appear to be true. If I do `VARIABLE=foo#bar` I get the value `foo` and not bar.

File: README.md (hunk 1 above)
Comment (limratana) [inline README.md]: I've double checked using both 

VAR = "SOMETHING#XXX" notation and 

VAR: 'SOMETHING#XXX" notation

**Pull request merged**